doc/source/_res/rn/useful-links.txt

```diff
@@ -0,0 +1,10 @@
+Useful links
+========================
+
+- The installation instructions for `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ are available.
+
+- This is the location for the `Percona Server downloads <https://www.percona.com/downloads/>`__.
+
+- The GitHub location for `Percona Server for MySQL <https://github.com/percona/percona-server>`__ 
```

Comment:
I would change it to smth like:
This is the location for `Percona Server for MySQL <https://github.com/percona/percona-server>`__  on GitHub.

Reply:
Updated.